gateway/response/response.go

```diff
@@ -105,9 +105,9 @@ func GetStatusCode(key string) int {
 	if status, ok := statusMap[key]; ok {
 		return status.code
 	}
-	return http.StatusOK
+	return http.StatusInternalServerError
```

Comment:
Since we only need to call this method for errors, we might as well rename it to `GetErrorStatusCode` to make it clear

Reply:
We make a call to this function with response.Ok key also. But only a few times compared to error responses.

Reply:
couldn't find such occurrences, can you please point me to them?